packages/taco/src/conditions/const.ts

```diff
@@ -4,6 +4,8 @@ export const USER_ADDRESS_PARAM = ':userAddress';
 
 export const ETH_ADDRESS_REGEXP = new RegExp('^0x[a-fA-F0-9]{40}$');
 
+export const CONTEXT_PARAM_REGEXP = new RegExp('^:[a-zA-Z0-9]+$');
```

Comment:
This regex would allow stuff like `:1234` as context variables, which I guess we don't want.

My suggestion:
```suggestion
export const CONTEXT_PARAM_REGEXP = new RegExp('^:[a-zA-Z]\\w*$');
```
This restricts the first character to letters, and for the rest it uses the word character class `\w` (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Character_class_escape#w)

Some simple tests:
```ts
const regex = new RegExp('^:[a-zA-Z]\\w*$');

console.log(regex.test(':piotr'));
console.log(regex.test(':hey_piotr'));
console.log(regex.test(':42'));

// Output
// > true
// > true
// > false
```

Reply:
Why wouldn't we want `:1234` as a condition parameter? Or `:1abc`?